code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -355,7 +366,7 @@
 		cached_gases[/datum/gas/water_vapor][MOLES] += burned_fuel / HYDROGEN_BURN_H2_FACTOR
 
 		energy_released += (FIRE_HYDROGEN_ENERGY_RELEASED * burned_fuel)
-		cached_results["fire"] += burned_fuel * 10
+		cached_results["fire"] += burned_fuel * 100
```

Comment:
Excuse me?

Reply:
had to increase this number by ten or the cold fire would die almost instantly, this doesn't change anything in the reaction itself

Reply:
Aight